src/sagemaker/analytics.py

```diff
@@ -516,6 +520,36 @@ def _reshape_metrics(self, metrics):
                     out["{} - {}".format(metric_name, stat_type)] = stat_value
         return out
 
+    def _reshape_input_artifacts(self, input_artifacts):
```

Comment:
why can't we just have _reshape_artifacts() which can be applied to both input/output-artifacts, so we dont need two unit tests over this too =)

Reply:
+1 would eliminate some redundancy

Reply:
thats true,